python/common/org/python/types/Str.java

```diff
@@ -1148,10 +1149,21 @@ private long toPositiveIndex(long index) {
     }
 
     @org.python.Method(
-            __doc__ = ""
+            __doc__ = "strip",
+            default_args = {"charsToBeRm"}
     )
-    public org.python.Object strip() {
-        throw new org.python.exceptions.NotImplementedError("strip() has not been implemented.");
+    public org.python.Object strip(org.python.Object charsToBeRm) {
+        String strim = "";
```

Comment:
@eliasdorneles ,I got an issue regarding rstrip function ..it fails in case of print('ABCBBAA'.rstrip('AB')) gives ABCBBAA , but it should be ABC.
but in my  implementation of rstrip it gives correct result. Should i submit PR.